c_src/eleveldb.cc

```diff
@@ -353,14 +357,15 @@ ERL_NIF_TERM parse_open_option(ErlNifEnv* env, ERL_NIF_TERM item, leveldb::Optio
         }
         else if (option[0] == eleveldb::ATOM_COMPRESSION)
         {
-            if (option[1] == eleveldb::ATOM_TRUE)
-            {
+            if (option[1] == eleveldb::ATOM_ON || option[1] == eleveldb::ATOM_TRUE
+                || option[1] == eleveldb::ATOM_SNAPPY )
                 opts.compression = leveldb::kSnappyCompression;
-            }
```

Comment:
My opinion on that same topic fluctuates.  Some days I agree with you and Iona.  Other days not.  No particular reason.  This was a case of that parent function getting too large and fewer lines seemed to make it seem clearer.  But I have heard (and even repeated) the argument that leaving the braces prepares the code for maintenance.

I will adjust if you feel strongly.